### PR TITLE
Fix KTX2 image loading fails

### PR DESCRIPTION
**Pull request by bimangle**
The **resource.url** has a timestamp suffix, causing **ktx2Regex.test()** return false. As a result, ktx2 image loading fails.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @bimangle!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by ggetz** [comment]
Thanks @bimangle! Would you be able to describe how the replicate the issue for testing?

**Comment by bimangle** [comment]
@ggetz 

Test url: [http://fs3.bimangle.net/cesium/t20221025-test-ktx2/index-online.html](http://fs3.bimangle.net/cesium/t20221025-test-ktx2/index-online.html)

Test data: [http://fs3.bimangle.net/cesium/t20221025-test-ktx2.zip](http://fs3.bimangle.net/cesium/t20221025-test-ktx2.zip)

t20221025-test-ktx2

**Comment by ggetz** [comment]
Thanks @bimangle! Could you please add an item to `CHANGES.md` describing what was fixed?

@sanjeetsuhag Would you be able to take a look and merge once ready?

File: Source/Scene/GltfImageLoader.js [hunk 1]
```diff
@@ -293,7 +293,7 @@ function loadImageFromBufferTypedArray(typedArray) {
 const ktx2Regex = /(^data:image\/ktx2)|(\.ktx2$)/i;
 
 function loadImageFromUri(resource) {
-  const uri = resource.url;
+  const uri = resource._url;
```
Comment (sanjeetsuhag) [inline Source/Scene/GltfImageLoader.js:296]:
```suggestion
  const uri = resource..getUrlComponent(false, true);
```

I think using using the [`resource.getUrlComponent`](https://github.com/CesiumGS/cesium/blob/main/Source/Core/Resource.js#L566) function would also do the same job without having to access a private variable. That's basically what the public getter for the `url` property is using under the hood anyway: https://github.com/CesiumGS/cesium/blob/4aac68a5d6b3603e13c3885b526b94a78b79f429/Source/Core/Resource.js#L469-L472

Comment (bimangle) [reply Source/Scene/GltfImageLoader.js:296]: It works. Your method is better.

**Comment by sanjeetsuhag** [comment]
@bimangle Aside from updating the `CHANGES.md`, it would also be helpful to add a test to ensure the URL query removal is working as expected.

**Comment by bimangle** [comment]
@ggetz @sanjeetsuhag  The CHANGES.md has been updated, and a test has been added.  Please take a look, thanks.

**Pull request merged**